### ISSUE TITLE
Improve is_day datapoint

### ISSUE THREAD

**Issue by Lawrence-Pet** [Hansstem/TDT4173-ML#8]
Prøve å lage sequensiell greie ala tidligere tensorflow documentation 

- Lag modell som kun ser å power og temporal ting ( en enkel en ) 
- Øke med flere og flere kolonner etterhvert
- Lag en form for statistikk på nøyaktigheten 
    - (Dele opp data settet i validation set og training set).
